<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>796</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>246</red>
        <green>246</green>
        <blue>246</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>246</red>
        <green>246</green>
        <blue>246</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>246</red>
        <green>246</green>
        <blue>246</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget#centralwidget{
	background-color: rgb(22, 22, 22);
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="buckets">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox, QAbstractItemView{
	background-color: #4f4f4f;
	color: #999999;
	selection-background-color: #999999;
	selection-color: #4f4f4f;

	border: 1px solid #76797C;
	border-radius: 2px;
	min-width: 75px;
}

QAbstractItemView {
	background-color: #232629;
    border-radius: 2px;
    border: 1px solid #76797C;
    selection-background-color: #3daee9;
}

QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 2em;

    border-left-width: -2px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 2px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow
{
    image: url(:/icons/downArrow.png);
}
QComboBox::down-arrow:on, QComboBox::down-arrow:hover,
QComboBox::down-arrow:focus
{
    image: url(:/icons/downArrow.png);
}</string>
    </property>
    <property name="title">
     <string>Buckets</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="deleteBucketsBTN">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Deletes the current Bucket&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Delete Bucket</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createBucketsBTN">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Creates a new Bucket&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;I am youuuu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Create Bucket</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bucketListCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load Buckets to Refresh List&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string extracomment="Buckets (Load Buckets to Refresh)"/>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshBucketsBTN">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get the current valid Buckets in your account&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Refresh Bucket List</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>371</width>
      <height>241</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
	background: rgb(240, 240, 240);
	border:  1px solid lightgray;
}
QTabBar::tab{
	background: rgb(240, 240, 240);
	border : 1px solid lightgray;
	width: 100px;
	height: 25px;
}
QTabBar::tab:selected{
	background: rgb(255, 255, 255);
	border : 1px solid lightgray;
	width: 100px;
	height: 25px;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="uploadTab">
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>271</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	border: 2px solid grey;
	border-radius: 5px;
	text-align: center;
}
QProgressBar::chunk{
	background-color: #05B8CC;
	color: black;
}</string>
     </property>
     <attribute name="title">
      <string>Upload</string>
     </attribute>
     <widget class="QPushButton" name="uploadFilesBTN">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload Files</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressAllFiles">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="addFileBTN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Files</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressCurrentFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
     <widget class="QListWidget" name="fileListWidg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="addFolderBTN">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transcribeTab">
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>241</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	border: 2px solid grey;
	border-radius: 5px;
	text-align: center;
}
QProgressBar::chunk{
	background-color: #05B8CC;
}
QListView{
	border: 2px solid grey;
	border-radius: 5px;
}</string>
     </property>
     <attribute name="title">
      <string>Transcribe</string>
     </attribute>
     <widget class="QPushButton" name="getBucketContentBTN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>List Contents</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subDownFolderTXT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Subtitle folder</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="subDownFolderBTN">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QListView" name="bucketFileListMod">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QPushButton" name="genSubtitleBTN">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Transcribe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="allTab">
     <property name="minimumSize">
      <size>
       <width>369</width>
       <height>212</height>
      </size>
     </property>
     <attribute name="title">
      <string>Auto-Transcribe</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bucketListCB</tabstop>
  <tabstop>refreshBucketsBTN</tabstop>
  <tabstop>deleteBucketsBTN</tabstop>
  <tabstop>createBucketsBTN</tabstop>
  <tabstop>fileListWidg</tabstop>
  <tabstop>addFileBTN</tabstop>
  <tabstop>addFolderBTN</tabstop>
  <tabstop>uploadFilesBTN</tabstop>
  <tabstop>bucketFileListMod</tabstop>
  <tabstop>subDownFolderTXT</tabstop>
  <tabstop>subDownFolderBTN</tabstop>
  <tabstop>getBucketContentBTN</tabstop>
  <tabstop>genSubtitleBTN</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
